src/Symfony/Component/Console/Application.php

```diff
@@ -596,7 +596,20 @@ public function find($name)
         $this->init();
 
         $aliases = [];
+
+        foreach ($this->commands as $command) {
+            foreach ($command->getAliases() as $alias) {
+                if (!$this->has($alias)) {
+                    $this->commands[$alias] = $command;
+                }
+            }
+        }
+
         $allCommands = $this->commandLoader ? array_merge($this->commandLoader->getNames(), array_keys($this->commands)) : array_keys($this->commands);
+
+        if ($this->has($name)) {
+            return $this->get($name);
+        }
```

Comment:
should be moved before the `$allCommands` assignment

Reply:
done